js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -290,10 +294,30 @@ const EditingController = modules.ViewController.inherit((function() {
             const changes = this.getChanges();
             const needReset = changes?.length;
             if(needReset) {
+                this._internalState = [];
                 this._silentOption('editing.changes', []);
             }
         },
 
+        _getInternalData: function(key) {
+            return this._internalState.filter(item => item.key === key)[0];
+        },
+
+        _addInternalData: function(params) {
+            const internalData = this._getInternalData(params.key);
+
+            if(internalData) {
+                return internalData;
```

Comment:
Add params assignment if internal data exists for case when data is refreshed for example.
`extend(internalData, params);`